integration/client-cli/admin-cli/src/main/java/org/keycloak/client/admin/cli/util/HttpUtil.java

```diff
@@ -262,6 +266,24 @@ public static void setTruststore(File file, String password) throws CertificateE
         sslsf = new SSLConnectionSocketFactory(theContext);
     }
 
+    public static void setSkipCertificateValidation() {
+        if (!tlsWarningEmitted.getAndSet(true)) {
```

Comment:
I believe it should be implemented this way. The `HttpUtil` has a bunch of static methods and I can easily imagine a situation where you call its methods in many threads. In such a case, you would see this error emitted twice (or even more times). Of course, I'd like to avoid such situation, therefore, using an `AtomicBoolean` is the easiest way to avoid it.